modules/api-explorer/examples/index.ts

```diff
@@ -0,0 +1,166 @@
+import http from "http";
+import PhenylRestApi from "phenyl-rest-api";
+import { createEntityClient } from "phenyl-memory-db";
```

Comment:
could you add a `TODO:` comment to change this to `@phenyl/memory-db` once memory-db is implemented?

Reply:
memory-db has been converted to TypeScript. Please use `@phenyl/memory-db` instead